### ISSUE TITLE
Email: Oracle feedback on WCAG 3.0 FPWD (2 - Scoring)

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#463]
[Comment from Email](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/0014.html): 

Scoring
While the example guidelines included a lot of detail, the information regarding scoring is throughout the document leading to a lot of confusion on how it would really work. Having an alternative to pass/fail is a requirement when it comes to applying WCAG to complex enterprise level products, but it needs to be done in a way that everyone can understand. Additionally, the methods used for scoring are complex since each outcome will have its own scoring scale. The impact to testing groups for the amount of data that must be recorded,  and tracking which scale is to be used on each outcome is going to lead to extensive time being spent on just tracking statistics of testing. We recommend a simplified scoring methodology be created to reduce this overhead.

**Comment by jspellman** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).

**Comment by cwadamsoforacle** [comment]
On behalf of Oracle, I am closing this issue.
